utxonursery.go

```diff
@@ -618,28 +622,7 @@ func (u *utxoNursery) reloadPreschool() error {
 	// hint from which to start our range for spend notifications.
 	for i := range psclOutputs {
 		kid := &psclOutputs[i]
-		chanPoint := kid.OriginChanPoint()
-
-		// Load the close summary for this output's channel point.
-		closeSummary, err := u.cfg.DB.FetchClosedChannel(chanPoint)
-		if err == channeldb.ErrClosedChannelNotFound {
-			// This should never happen since the close summary
-			// should only be removed after the channel has been
-			// swept completely.
-			utxnLog.Warnf("Close summary not found for "+
-				"chan_point=%v, can't determine height hint"+
-				"to sweep commit txn", chanPoint)
-			continue
-
-		} else if err != nil {
-			return err
-		}
-
-		// Use the close height from the channel summary as our height
-		// hint to drive our spend notifications, with our confirmation
-		// depth as a buffer for reorgs.
-		heightHint := closeSummary.CloseHeight - u.cfg.ConfDepth
```

Comment:
I don't think it adds much value, as this is only applied at restart, and reorgs are not really handled in other cases other than waiting for `ConfDepth` before acting on it.

So I decided to remove it, as it simplifies the logic, and if we really want to be reorg safe, the non-restart case must also be handled.

Reply:
Is it really worth risking the possibility of doing an entire chain rescan to avoid one db read on startup? It almost seems like we can leave this as is, and just pass the broadcast height in-memory to avoid having two different serialization formats.